### PR TITLE
NO-ISSUE - Update SNO image used for testing

### PR DESCRIPTION
**Pull request by omertuc**
*Work in progress*

Upgrading the SNO image to a more recent 4.8.0 nightly.

Since it's not hosted on svc.ci, we don't need it as a public registry anymore, so I removed svc.ci as a public registry

### CODE REVIEW

File: hack/get_ocp_versions_for_testing.sh [hunk 1]
```diff
@@ -4,7 +4,8 @@ set -euo pipefail
 
 __dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 PROJECT_ROOT_DIR="$(cd "$(dirname "${__dir}")" && pwd)"
-OPENSHIFT_SNO_IMAGE="registry.svc.ci.openshift.org/sno-dev/openshift-bip:0.6.0"
+OPENSHIFT_SNO_IMAGE="quay.io/openshift-release-dev/ocp-release-nightly:4.8.0-0.nightly-2021-03-16-221720"
+"
```
Comment (osherdp) [inline hack/get_ocp_versions_for_testing.sh:8]: ?

Comment (omertuc) [reply hack/get_ocp_versions_for_testing.sh]: "

**Review by osherdp — APPROVED** [review]
/lgtm

**Comment by omertuc** [comment]
@eranco74 expanded on this PR in https://github.com/openshift/assisted-service/pull/1349

**Pull request closed without merging**